File: jeorg-camel-app-atm/src/main/java/com/jesperancinha/atm/finder/service/payload/response/ATMAddress.java
package com.jesperancinha.atm.finder.service.payload.response;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Getter;
import lombok.Setter;

/**
 * Created by joaofilipesabinoesperancinha on 28-07-16.
 */
@Getter
@Setter
@JsonIgnoreProperties(ignoreUnknown = true)
public class ATMAddress {

    @JsonProperty("street")
    private String street;

    @JsonProperty("housenumber")
    private String housenumber;

    @JsonProperty("postalcode")
    private String postalcode;

    @JsonProperty("city")
    private String city;

    @JsonProperty("geoLocation")
    private ATMGeoLocation geoLocation;
}

File: jeorg-camel-app-atm/src/main/kotlin/com/jesperancinha/atm/finder/service/payload/response/ATMAddress.kt
package com.jesperancinha.atm.finder.service.payload.response

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import com.fasterxml.jackson.annotation.JsonProperty
import lombok.Getter
import lombok.Setter

/**
 * Created by joaofilipesabinoesperancinha on 28-07-16.
 */
@Getter
@Setter
@JsonIgnoreProperties(ignoreUnknown = true)
class ATMAddress {
    @JsonProperty("street")
    private val street: String? = null

    @JsonProperty("housenumber")
    private val housenumber: String? = null

    @JsonProperty("postalcode")
    private val postalcode: String? = null

    @JsonProperty("city")
    private val city: String? = null

    @JsonProperty("geoLocation")
    private val geoLocation: ATMGeoLocation? = null
}